test/yarp/newline_test.rb

```diff
@@ -36,12 +36,7 @@ def assert_newlines(base, relative)
 
       result = YARP.parse_file(filepath)
       assert_empty result.errors
-
-      result.mark_newlines!
-      visitor = NewlineVisitor.new(result.source)
```

Comment:
The NewlineVisitor class above can be removed then, right?

Reply:
It was in a subsequent commit